src/ImGuiEx.cpp

```diff
@@ -104,6 +123,20 @@ float ImGuiEx::StatsEntry(std::string_view pLeftText, std::string_view pRightTex
 	ImGui::EndGroup();
 	ImGui::PopStyleColor();
 
-	// window padding - inner spacing - left text - item spacing * 2 - right text - inner spacing - window padding
-	return leftTextSize.x + rightTextSize.x + ImGui::GetStyle().ItemSpacing.x * 2.0f + ImGui::GetStyle().ItemInnerSpacing.x * 2.0f + ImGui::GetCurrentWindowRead()->WindowPadding.x * 2.0f;
-}
\ No newline at end of file
+	// index number - window padding - inner spacing - left text - item spacing * 3 - right text - inner spacing - window padding
+	return indexNumberSize.x + leftTextSize.x + rightTextSize.x + professionTextSize.x + ImGui::GetStyle().ItemSpacing.x * 4.0f + ImGui::GetStyle().ItemInnerSpacing.x * 2.0f + ImGui::GetCurrentWindowRead()->WindowPadding.x * 2.0f;
```

Comment:
the itemspacing etc. is only relevant if they are being used, right? Whereas now you unconditionally add them

Perhaps add them to the size calculation in the if cases above, instead? Should make it a bit more readable too, where they are coming from

Reply:
Also, did you confirm that this is an ItemSpacing? I got the original things here by reading through imgui source code. Would appreciate if you can look through your newly added ones as well to make sure the size calculation is right.